CHANGES

```diff
@@ -10,6 +10,15 @@ Version 1.3.4-dev (development of upcoming release)
 * Fix bug: Unit test fails on some machines due to warning "Ignoring XDG_SESSION_TYPE=wayland on Gnome..." (#1429)
 * Fix bug: Generation of config-manpage caused an error with Debian's Lintian (#1398).
 * Fix bug: Return empty list in smartRemove (#1392, Debian Bug Report 973760)
+* Fix bug: Taking a snapshot reports `rsync` errors now even if no snapshot was taken (#1491)
+* Fix bug: takeSnapshot() recognizes errors now by also evaluating the rsync exit code (#489)
+           Fixes related problem: Killing `rsync` was not handled gracefully (by ignoring the rsync exit code)
+* Fix bug: The error user-callback is now always called if an error happened during taking a snapshot (#1491)
+* Improvement: Introduce new error codes for the "error" user callback (as part of #1491):
```

Comment:
Based on changelog standards I would name this a "Feature".

Reply:
Do you still found the link to the changelog standards (I have lost oversight here).

I would like to add it at the top of the changelog file keep it "in sight" whenever I change the changelog...

Reply:
In my [own projects](https://codeberg.org/buhtz/buhtzology) where I also decided to write [markdown changelog](https://codeberg.org/buhtz/buhtzology/src/branch/develop/CHANGELOG.md) I do use
 - https://common-changelog.org
 - https://keepachangelog.com
 - and slightly related https://www.conventionalcommits.org

I didn't want to bother you folks with that details.

Reply:
Fixed